pages.uk/common/git-annotate.md

```diff
@@ -0,0 +1,18 @@
+# git annotate
+
+> Показує хеш коміту і останнього автора на кожному рядку у файлі.
+> Дивіться `git blame`, якому варто віддати перевагу над `git annotate`.
```

Comment:
```suggestion
> Дивіться на `git blame`, якому варто віддати перевагу над `git annotate`.
```

Reply:
I've looked for any rules or at least examples of usage. Generally, it is used mostly with physical objects (e.g., Look at this car), but not with intangible ones (e.g., Watch a movie). Also, the English version of this doc used "See" and not "See at". So, I preferred to leave it as is.
But I caught there another issue :)

Anyway, thank you for your notice. I leave it unresolved while fixing that other place.

Reply:
Improved translations in other places.